decent-dao/fractal-interface
# UI | Nodes Page

## Description
Adds Node Page UI. Should be easy to switch out mocked data for the real thing after when #667  is merged


## Notes

- Extracts DAOInfo's UI into a reusable component that can be used by both the Dashboard and Nodepage components
- Creates NodesLines component to show connections
- Combines Mikes ManageDAO menu into new component (applied a few fixes as well)
- Adds subDAO number UI
- Adds option function for create a new subDAO (route not setup yet)



## Issue / Notion doc (if applicable)



## Testing
- Navigate to a DAO
- View DAO Dashboard Info card ensure that data is properly loaded
- Navigate to Nodes Page using the sidebar link
- View DAO Nodes Page UI that it matches Figma designs https://www.figma.com/file/uRV3Nfowdzt5CknW1FgUkQ/%F0%9F%93%9A-Fractal-Design-System-%7C-Base-Components-%7C-October-2022?node-id=2556%3A23342&t=m3VWcHiKyn6nyNdG-4


## Screenshots (if applicable)
### DAO Dashboard
![image](https://user-images.githubusercontent.com/38386583/201999645-e0c0c156-154c-4802-956e-b789d54c9f6c.png)


### Nodes Pages (Parent + Current + Children of Current (collasped))
![localhost_3000_(Desktop) (1)](https://user-images.githubusercontent.com/38386583/202000113-5f0dfca1-18b9-4b73-8a33-32378f05a171.png)

### Nodes Pages (Parent + Current + Children of Current (expanded))
![localhost_3000_(Desktop) (2)](https://user-images.githubusercontent.com/38386583/202000190-2b3254ab-387e-400c-974e-85e1c091756a.png)

### Nodes Page (current DAO only, menu open)
![localhost_3000_(Desktop) (3)](https://user-images.githubusercontent.com/38386583/202000585-ec0440ad-d16f-455e-b4c2-e3e24e85047d.png)


### Nodes Page (Highlights Current DAO)
![localhost_3000_(Desktop) (4)](https://user-images.githubusercontent.com/38386583/202061508-51cec2ed-bd30-4c4d-8ba6-6f78463ad941.png)
